statsd/statsd_test.go

```diff
@@ -540,9 +540,9 @@ func TestSendMsgUDP(t *testing.T) {
 		t.Error("Expected error to be returned if message size is bigger than MaxUDPPayloadSize")
 	}
 
-	longMsg := strings.Repeat("x", MaxUDPPayloadSize)
+	message := "test message"
```

Comment:
Not sure if there was a legit reason to send messages at the `MaxUDPPayloadSize`? I believe they were maybe trying to test there was no unexpected fragmentation of the statsd payload unless `len(message) > MaxUDPPayloadSize`?

I'm really not sure, just food for thought.

Reply:
I suspect it was for that reason, problem is this test would depend on the kernel settings of the hosts running the tests, IMO a bit out of scope for a unit test so I just lowered the message to a size that's not supposed to cause fragmentation.